docs/welcome/key-metrics-guide.md

```diff
@@ -0,0 +1,69 @@
+---
+id: key-metrics-guide
+title: "Key Metrics Guide"
+sidebar_label: "Key Metrics Guide"
+keywords:
+  - "metrics"
+  - "openSauced"
+  - "OpenSauced metrics guide"
+  - "Key metrics guide"
+  - "Open Source metric guide"
+---
+
+Welcome to the Key Metrics Guide, where we define the main metrics used on our platform to help you better understand them.
+
+## OSCR
+
+[OSCR](../glossary/#oscr) stands for Open Source Contributor Rating. It's a way to measure and rate the impact of your open-source contributions over 90 days.
+
+This helps you understand the quality and frequency of your contributions. It's a good way to measure your open source activities and takes into account many factors.
+
+You can read more about [what it is and how to improve it](../../opensauced-guides/oscr-score-guide/oscr-guide/).
+
+## Contributor Confidence
+
+[Contributor Confidence](../glossary/#contributor-confidence) is a metric that measures how likely contributors who have interacted with a repository via stars or forks are to contribute in some way. It's measured as a percentage calculated over a specified time range.
+
+This helps you evaluate whether a repository you are interested in provides a good environment for new contributors. A positive score implies that the project is welcoming to new contributors or that those who interact with the project are willing to support the project through contributions.
```

Comment:
Let's move this to third person as well: This helps users to understand if the project is welcoming to new contributors.